jwql/utils/preview_image.py

```diff
@@ -821,3 +843,2599 @@ def nan_to_zero(image):
     nan = np.isnan(image)
     image[nan] = 0
     return image
+
+
+class Level3PreviewImage():
+    """An object for generating and saving a preview image from level 3 files.
+    Used by``generate_preview_images``.
+    """
+    def __init__(self, filename, data_type=None, maxsize=8, preview_output_directory=None,
```

Comment:
If we could make docstrings for the parameters here, I think that would be helpful!

Reply:
done